app/pipedrive/api.py

```diff
@@ -52,13 +76,31 @@ async def create_or_update_organisation(company: Company) -> Organisation:
             await pipedrive_request(f'organizations/{company.pd_org_id}', method='PUT', data=hermes_org_data)
             app_logger.info('Updated org %s from company %s', company.pd_org_id, company.id)
     else:
-        # if company is not on pipedrive, create it
+        # if company is not linked to pipedrive, compare the org name with all orgs in pipedrive
+        # if there is a match, link the company to the org and update the org
+        if company.tc2_cligency_id:
+            tc2_cligency_url = f'{settings.tc2_base_url}/clients/{company.tc2_cligency_id}/'
```

Comment:
You need to explain that this tc2_cligency_url was a field in Hubspot and that's where this data may have come from.

We should probably have a TODO that says we should search on more items in the future.